ui/src/app/pages/login/account-creation/account-creation-success.tsx

```diff
@@ -54,51 +54,49 @@ export class AccountCreationSuccess
 
   render() {
     return <React.Fragment>
-      <div style={{padding: '3rem 3rem 0 3rem', marginLeft: '-0.5rem', marginRight: '-0.5'}}>
-        <BolderHeader>
+      <div style={{padding: '3rem 3rem 0 3rem', marginLeft: '-0.5rem', marginRight: '-0.5', width: '25rem'}}>
+        <PageHeader>
           Congratulations!
-        </BolderHeader>
-        <div>
-          <SmallHeader style={{color: colors.primary}}>
+        </PageHeader>
+
+        <TextColumn>
+          <div style={{fontSize: '16px', marginTop: '.25rem'}}>
             Your <i>All of Us</i> research account has been created!
-          </SmallHeader>
-        </div>
-        <div>
-          <Header style={{fontWeight: 400, textColor: colors.primary}}>
-            Your new account
-          </Header>
-        </div>
-        <div style={{whiteSpace: 'nowrap'}}>
-          <Header style={{fontWeight: 400, marginTop: '0.5rem'}}>
+          </div>
+        </TextColumn>
+
+        <Divider verticalMargin='1rem' style={{width: '100%'}}/>
+
+        <TextColumn style={{fontSize: '20px', lineHeight: '30px'}}>
+          <div>
+            Your new research workbench account
+          </div>
+          <div>
             {this.props.profile.username}
-          </Header>
-        </div>
-        <div>
-          <Header style={{marginTop: '.5rem', fontWeight: 400}}>
+          </div>
+          <div>
             is hosted by Google.
-          </Header>
-        </div>
-        <div>
-          <SmallHeader style={{color: colors.primary}}>
-            Check your contact email for instructions on getting started.
-          </SmallHeader>
-        </div>
-        <div>
-          <SmallHeader style={{color: colors.primary}}>
-            Your contact email is: {this.state.contactEmail}
-          </SmallHeader>
-        </div>
-        <div style={{paddingTop: '0.5rem'}}>
-          <button style={styles.buttonLinkStyling}
+          </div>
+          <div style={{marginTop: '1rem'}}>
+            Check your contact email for
```

Comment:
I think this one is probably okay if the line break isn't right where it is in the comp. (The other one is important because it puts the username all on one line on its own.)

Reply:
True, I was thinking it could go either way but I figured I'd just follow the mocks.